### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AmandaL-IS**
Update README.md to reflect new npm installation processes and npm tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,50 +10,84 @@
...
-The xGraph command line interface accelerates the building and running
-of xGraph Systems.
 
----
-#### Getting Started
-You will find all the necessary installation files in the `.\bin\`
-directory. To install the xgraph CLI, follow the instructions for your
-operating system found below.
+The xGraph package comes with two main features: the xGraph core and the xGraph 
+Command Line Interface.
```
Comment (valyrie97) [inline README.md]: I would only include references to the Command Line utility here. The only people who need to care about nexus/genesis are those who are using xgraph programmatically.

As well, calling the files on their own is also no longer supported.

File: README.md [hunk 2]
```diff
@@ -10,50 +10,84 @@
...
 
-Additionally, you will have to add the xgraph path variable to your
-systems environment variable. This can be done for a single session
-through the command line, or you can set the environment variable
-permanently through windows settings.
+### xGraph Core
+The xGraph core architecture handles compiling, instantiating and communication 
+between xGraph systems. The xGraph core consists of Genesis and Nexus. Genesis 
+handles compilation of xGraph systems, while Nexus handles instantiation of and 
+communication between xGraph systems.
```
Comment (valyrie97) [inline README.md]: Like above, Users dont need to know about Nexus/Genesis.

File: README.md [hunk 3]
```diff
@@ -10,50 +10,84 @@
...
+In your command line tool, use the following command to install xGraph core 
+globally using the npm package installer.
+```
+npm install -g xgraph
+```
 
-##### For Linux:
-Simply unpack the installation file (.tar or .gz) and restart your terminal
-session.
+If you are on a unix machine, you may have to use the `sudo` prefix.
```
Comment (valyrie97) [inline README.md]: I would add (Like Ubuntu or macOS) just to be more clear and user friendly

File: README.md [hunk 4]
```diff
@@ -10,50 +10,84 @@
...
+```
+sudo npm install -g xgraph
+```
 
+##### Install using npm package builder
```
Comment (valyrie97) [inline README.md]: I would retitle to something more along the lines of "Building the Standalone with npm", as the actual build process does not use npm, and "npm package builder" isnt a thing. `npm run build` is just a macro for calling the build script with node. the actual library doing the compilation is [nexe](https://www.npmjs.com/package/nexe). as well, the resulting executables dont require node or npm to be installed afterwards

**Review by valyrie97 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AmandaL-IS) [reply README.md]: I disagree. This is the README.md for the core, so it is what shows up on our git homepage, so we should explain what it is. Once we have a website, we can provide better user-end instructions.

What do you mean calling files on their own is no longer supported?

File: README.md (hunk 4 above)
Comment (AmandaL-IS) [reply README.md]: https://docs.npmjs.com/cli/build npm build, builds a package... this isn't what we are using?

Comment (valyrie97) [reply README.md]: no

File: README.md (hunk 1 above)
Comment (valyrie97) [reply README.md]: I agree, that we should provide all the relevant information here about what xgraph is, but "Nexus" and "Genesis" arent really public facing anymore, they're merely an implementation detail of the command line utility. The end-user will never even see the words nexus or genesis if we've done our job well.

I mean exactly that, you can no longer call node directly on nexus or genesis. they are not executable files. they are effectively two libraries that xgraph uses to create systems

File: README.md (hunk 4 above)
Comment (valyrie97) [reply README.md]: in [package.json](https://github.com/IntrospectiveSystems/xGraph/blob/release/package.json#L7) we override that functionality.

as well, you dont show how to actually install anything here, only build it. installing the standalone requires you to either build an installer as well (which is currently not supported for the end user), or you need to put the resulting executable in a folder that is in your path. for example on linux, `/usr/bin`

**Review by valyrie97 — APPROVED** [review]

**Pull request merged**